tools/mapping_to_ucsc/.shed.yml

```diff
@@ -1,4 +1,9 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/mapping_to_ucsc
-owner: devteam
+categories: []
```

Comment:
This category would be better called Sequence Alignment, but that's what we have.